### ISSUE TITLE
Build fails when the footnotes is removed, even if there are no footnotes

### ISSUE THREAD

**Issue by thesephist** [webdesigndecal/mark2html#6]
Taking out the footnotes section markers causes an undefined-method error or something, even if there are no footnotes. This should probably not happen, so we can remove or hide footnotes box for assignments without footnotes.

**Comment by sethlu** [comment]
@thesephist Can you reproduce the issue on a separate file?

**Comment by thesephist** [comment]
Yeah it was an issue I noticed with the last lab (6, I think). There are workaround, but it was more of a convenience bug. Gonna try to fix it tonight.
